app/assets/javascripts/static.js

```diff
@@ -3,31 +3,23 @@
 
 // Replaces the magnifying glass in the search button with a rotating wait icon
 // See changes in customers.css.scss too, as well as /catalog and /home html.
-// $(document).on("click",'button#search',function(){
-//   // add validation here
-//   // show search in progress
-//   $("div.search span#not-searching").addClass("hidden");
-//   $("div.search span#now-searching").removeClass("hidden");
-//   $("body").css("cursor", "progress");
-// });
 
 $( document ).ready(function() {
 
-    // if $('input#q').length > 0 {
-        $("div.search span#not-searching").removeClass("hidden");
-        $("div.search span#now-searching").addClass("hidden");
-        $("body").css("cursor", "auto");
-    // }
-
+    $("div.search span#not-searching").removeClass("hidden");
+    $("div.search span#now-searching").addClass("hidden");
+    $("body").css("cursor", "auto");
+    
+    set_dates("div.search ");
+    event.preventDefault();
 
-  // Replaces the magnifying glass in the search button with a rotating wait icon
-  // See changes in customers.css.scss too, as well as /catalog and /home html.
-  $(document).on("click", 'button#search', function(){
 
+  $(document).on("click", 'div.search form button#search', function(){
```

Comment:
Yep, you're right. I was thinking of renaming the facet button to button#search when I added that. But in the end I didn't rename it.